src/css/application.css

```diff
@@ -471,19 +471,29 @@ body {
   height: 40vh;
 }
 
-.console__entry,
+.console__entry-active,
 .console__input {
   margin-bottom: 0.5em;
 }
 
-.console__entry {
+.console__entry-active {
```

Comment:
In BEM, we should always have a base block or element (in this case `console__entry`) alongside any necessary modifiers (in this case `console__entry_active`). This would prove practically useful as well as there would no longer be a need to duplicate the 4 pixel margin, for instance.

I would also lean toward just having a single `_inactive` modifier, with the base element definition having the active look.

Also this particular flavor of BEM uses a single underscore, rather than a dash, for modifiers.